src/commander/bookmarks.rs

```diff
@@ -134,14 +134,15 @@ impl Commander {
                 None => BookmarkLine::Unparsable(line_colored.to_owned()),
             })
             .sorted_by(|a, b| {
-                if let (
-                    BookmarkLine::Parsed { bookmark: a, .. },
-                    BookmarkLine::Parsed { bookmark: b, .. },
-                ) = (a, b)
-                {
-                    b.timestamp.cmp(&a.timestamp)
-                } else {
-                    Ordering::Equal
+                use BookmarkLine::*;
+
+                match (a, b) {
+                    (Parsed { bookmark: a, .. }, Parsed { bookmark: b, .. }) => {
+                        b.timestamp.cmp(&a.timestamp)
+                    }
+                    (Parsed { .. }, Unparsable(..)) => Ordering::Less,
+                    (Unparsable(..), Parsed { .. }) => Ordering::Greater,
```

Comment:
Just curious, can we swap it? Does it just have to be stable, or does it depend on something?

Reply:
Maybe add a small comment why ordering is required? About crash